[2024-01-15T15:11:52.441-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cities_novo.df scheduled__2024-01-15T18:09:41.422047+00:00 [queued]>
[2024-01-15T15:11:52.445-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cities_novo.df scheduled__2024-01-15T18:09:41.422047+00:00 [queued]>
[2024-01-15T15:11:52.445-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-01-15T15:11:52.458-0300] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): df> on 2024-01-15 18:09:41.422047+00:00
[2024-01-15T15:11:52.461-0300] {standard_task_runner.py:57} INFO - Started process 68296 to run task
[2024-01-15T15:11:52.463-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cities_novo', 'df', 'scheduled__2024-01-15T18:09:41.422047+00:00', '--job-id', '1093', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpnigaeary']
[2024-01-15T15:11:52.464-0300] {standard_task_runner.py:85} INFO - Job 1093: Subtask df
[2024-01-15T15:11:52.494-0300] {task_command.py:416} INFO - Running <TaskInstance: cities_novo.df scheduled__2024-01-15T18:09:41.422047+00:00 [running]> on host desktop
[2024-01-15T15:11:52.543-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='datasetpibic@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='cities_novo' AIRFLOW_CTX_TASK_ID='df' AIRFLOW_CTX_EXECUTION_DATE='2024-01-15T18:09:41.422047+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-15T18:09:41.422047+00:00'
[2024-01-15T15:11:56.629-0300] {logging_mixin.py:154} INFO - Called | 1705342316.6268501 | df
[2024-01-15T15:11:56.636-0300] {logging_mixin.py:154} INFO - SAVING FILE df
[2024-01-15T15:11:56.636-0300] {python.py:194} INFO - Done. Returned value was: None
[2024-01-15T15:11:56.639-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=cities_novo, task_id=df, execution_date=20240115T180941, start_date=20240115T181152, end_date=20240115T181156
[2024-01-15T15:11:56.722-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-15T15:11:56.731-0300] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
